pipeline/tests/powdr_std.rs

```diff
@@ -422,18 +424,35 @@ fn fp4() {
     evaluate_function(&analyzed, "std::math::fp4::test::add", vec![]);
     evaluate_function(&analyzed, "std::math::fp4::test::sub", vec![]);
     evaluate_function(&analyzed, "std::math::fp4::test::mul", vec![]);
+    evaluate_function(&analyzed, "std::math::fp4::test::square", vec![]);
     evaluate_function(&analyzed, "std::math::fp4::test::inverse", vec![]);
 
     let analyzed = std_analyzed::<Bn254Field>();
     evaluate_function(&analyzed, "std::math::fp4::test::add", vec![]);
     evaluate_function(&analyzed, "std::math::fp4::test::sub", vec![]);
     evaluate_function(&analyzed, "std::math::fp4::test::mul", vec![]);
+    evaluate_function(&analyzed, "std::math::fp4::test::square", vec![]);
     evaluate_function(&analyzed, "std::math::fp4::test::inverse", vec![]);
 
     let analyzed = std_analyzed::<BabyBearField>();
     evaluate_function(&analyzed, "std::math::fp4::test::add", vec![]);
     evaluate_function(&analyzed, "std::math::fp4::test::sub", vec![]);
     evaluate_function(&analyzed, "std::math::fp4::test::mul", vec![]);
+    evaluate_function(&analyzed, "std::math::fp4::test::square", vec![]);
+    evaluate_function(&analyzed, "std::math::fp4::test::inverse", vec![]);
+
+    let analyzed = std_analyzed::<Mersenne31Field>();
+    evaluate_function(&analyzed, "std::math::fp4::test::add", vec![]);
+    evaluate_function(&analyzed, "std::math::fp4::test::sub", vec![]);
+    evaluate_function(&analyzed, "std::math::fp4::test::mul", vec![]);
+    evaluate_function(&analyzed, "std::math::fp4::test::square", vec![]);
+    evaluate_function(&analyzed, "std::math::fp4::test::inverse", vec![]);
+
+    let analyzed = std_analyzed::<KoalaBearField>();
+    evaluate_function(&analyzed, "std::math::fp4::test::add", vec![]);
+    evaluate_function(&analyzed, "std::math::fp4::test::sub", vec![]);
+    evaluate_function(&analyzed, "std::math::fp4::test::mul", vec![]);
+    evaluate_function(&analyzed, "std::math::fp4::test::square", vec![]);
     evaluate_function(&analyzed, "std::math::fp4::test::inverse", vec![]);
```

Comment:
We should be testing with these fields, because we'd also be using Fp4 for those.